2-mandatory/4-people-i-know.js

```diff
@@ -378,13 +378,19 @@ First, I want you to find all of my friends who are 35 or older.
 */
 
 let thirtyFiveOrOlder = [];
+friends.map((friend) => {
+  if (friend.age >= 35) thirtyFiveOrOlder.push(friend.name);
+});
 
 /*
 3) Find the email address
 Next, I want you to find all of my friends who work for "POWERNET" and then store their emails in the array below
 */
 
 let powerNetEmails = [];
+friends.map((friend) => {
+  if (friend.company === "POWERNET") powerNetEmails.push(friend["email"]);
+});
```

Comment:
Here a suggestion using array Methods
```suggestion
powerNetEmails = friends
  .filter((friend) => friend.company == "POWERNET")
  .map((friend) => friend.email);
```

Reply:
Thanks @SaadiaELF great idea.